src/devices/src/virtio/net/device.rs

```diff
@@ -543,8 +542,7 @@ impl Net {
     pub fn process_tap_rx_event(&mut self) {
         METRICS.net.rx_tap_event_count.inc();
         if self.queues[RX_INDEX].is_empty(&self.mem) {
-            error!("The RX queue is empty, there is no available buffer.");
-            METRICS.net.event_fails.inc();
+            METRICS.net.no_rx_avail_buffer.inc();
```

Comment:
You need to increment this metric in _Line 225_ too.

```rust
    fn rx_single_frame(&mut self) -> bool {
        let rx_queue = &mut self.queues[RX_INDEX];
        let mut next_desc = rx_queue.pop(&self.mem);
        if next_desc.is_none() {
            // No buffers available <---------------------- increment metric
            return false;
        }
        ...
    }
```

Reply:
This is true. I'll increment the no_rx_avail_buffer metric here, too.